java-security-test/src/main/java/com/sap/cloud/security/test/SecurityTestRule.java

```diff
@@ -158,6 +160,27 @@ public SecurityTestRule setPort(int port) {
 		return this;
 	}
 
+	/**
+	 * Same as {@link SecurityTestRule#setKeys(RSAKeys)} except you can pass in the
```

Comment:
I would remove setKeys(RSAKey), its no longer needed.

Reply:
You'r right, externally this is probably not needed anymore. Should we declare it  package protected since it is still used inside the test of the rule itself. Or would you rather use static keys for this test as well?

Reply:
i would remove it and have only a test with the key files.